SDK/AppCenterData/Microsoft.AppCenter.Data.Shared/Data.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Microsoft.AppCenter.Data
+{
+    public partial class Data
+    {
+        /// <summary>
+        /// Change the base URL used to make API calls.
+        /// </summary>
+        /// <param name="apiUrl">API base URL.</param>
+        public static void SetApiUrl(string apiUrl)
+        {
+            PlatformSetApiUrl(apiUrl);
+        }
+
+        /// <summary>
+        /// Check whether the Data service is enabled or not.
+        /// </summary>
+        /// <returns>A task with result being true if enabled, false if disabled.</returns>
+        public static Task<bool> IsEnabledAsync()
+        {
+            return PlatformIsEnabledAsync();
+        }
+
+        /// <summary>
+        /// Enable or disable the Data service.
+        /// </summary>
+        /// <returns>A task to monitor the operation.</returns>
+        public static Task SetEnabledAsync(bool enabled)
+        {
+            return PlatformSetEnabledAsync(enabled);
+        }
+
+        /// <summary>
+        /// Read a document.
```

Comment:
Need param and return documentation for all methods.

Reply:
Android SDK needs it too 😉